docs/markdown/Python-module.md

```diff
@@ -12,6 +12,15 @@ authors:
 This module provides support for finding and building extensions against
 python installations, be they python 2 or 3.
 
+If you want to build and package Python extension modules using tools
+compatible with [PEP-518](https://peps.python.org/pep-0518/), check out
```

Comment:
Probably you mean PEP 517 (which specifies build-system.build-backend and the `build_wheel()` API, whereas 518 defines build-system.requires).